shopsafe-backend/src/main/java/com/google/sps/data/CheckInStats.java

```diff
@@ -122,53 +123,54 @@ public long getCheckInCount() {
       Date currentDate = (Date) ratingEntity.getProperty("Date");
       Date nextDate = (Date) nextEntity.getProperty("Date");
 
-      // Check if next entitiy is apart of the same day
+      // Check if next entitiy is apart of the same day.
       int r1day = currentDate.getDay();
       int r2day = nextDate.getDay();
       endDay = !(r1day == r2day);
 
+      // If end of day, get accumulated ratings and add them.
       if (endDay) {
-        // get accumulated ratings
         ArrayList<DataPoint> maskRatings = returnMap.get("mask");
-        ArrayList<DataPoint> hygieneRatings = returnMap.get("hygiene");
-        ArrayList<DataPoint> lineRatings = returnMap.get("line");
-        ArrayList<DataPoint> busyRatings = returnMap.get("busy");
-
-        // add average
         maskRatings.add(new DataPoint(maskSum / (ratingNum + 1.0), currentDate));
+        ArrayList<DataPoint> hygieneRatings = returnMap.get("hygiene");
         hygieneRatings.add(new DataPoint(hygieneSum / (ratingNum + 1.0), currentDate));
+        ArrayList<DataPoint> lineRatings = returnMap.get("line");
         lineRatings.add(new DataPoint(lineSum / (ratingNum + 1.0), currentDate));
+        ArrayList<DataPoint> busyRatings = returnMap.get("busy");
         busyRatings.add(new DataPoint(busySum / (ratingNum + 1.0), currentDate));
```

Comment:
im guessing this came from the styling guide : (

Reply:
Yeah, it said the distance between declaration and usage was too far.